static/docker-compose.yml

```diff
@@ -5,41 +5,40 @@ services:
       update_config:
         parallelism: 1
     depends_on:
-    - plextracdb
-    - redis
-    - postgres
+      - plextracdb
```

Comment:
looks like my format on save put some work in on this file.. I can revert the formatting changes if this is a concern

Reply:
Can we remove the formatting here? I'm slightly concerned the indent will cause issues with file